index.html

```diff
@@ -14,20 +14,23 @@ <h1>Sample Web Store</h1>
         <nav>
             <ul>
                 <li>
-                    <a href="#">Home</a>
```

Comment:
All the red anchors in this section I am not seeing in my VS. it shows up as the one below this and same with product 1 and even the product that I worked on shows up as just shoes not 2 different lines that are tying to the same thing. Hope this makes sense. Im not sure how to fix it so it only shows the green part.